bundles/org.eclipse.ui.editors/src/org/eclipse/ui/internal/editors/text/TextEditorMessages.properties

```diff
@@ -17,6 +17,8 @@ EditorsPlugin_internal_error=Internal Error
 
 TextEditorPreferencePage_displayedTabWidth=Displayed &tab width:
 TextEditorPreferencePage_lineSpacing=Line &spacing (extra % of font height):
+TextEditorPreferencePage_useFindReplaceOverlay=Use find/replace overlay
+TextEditorPreferencePage_showFindReplaceOverlayAtBottom=Display find/replace overlay at bottom of editor
```

Comment:
It would be good here as well to assign a shortcut value. "d" does not seem to be used on the page yet, so using "&Display ..." should work.